env/WCOSS2.env

```diff
@@ -43,11 +43,29 @@ elif [ $step = "waveinit" -o $step = "waveprep" -o $step = "wavepostsbs" -o $ste
 
 elif [ $step = "anal" ]; then
 
+    # OMP settings
+    export OMP_PLACES=cores
+    export OMP_NUM_THREADS=$nth_anal
+    export MP_EAGER_LIMIT=165536
+    export MP_COREFILE_FORMAT=lite
+    export MP_EUIDEVELOP=min
+    export MP_EUIDEVICE=sn_all
+    export MP_EUILIB=us
+    export MP_MPILIB=mpich2
+    export MP_LABELIO=yes
+    export MP_SINGLE_THREAD=yes
+    export MP_USE_BULK_XFER=yes
+    export MP_SHARED_MEMORY=yes
+    #export MPICH_ALLTOALL_THROTTLE=0
+    export MP_COLLECTIVE_OFFLOAD=no
+    export KMP_STACKSIZE=1024m
+    export FI_OFI_RXM_SAR_LIMIT=3145728
```

Comment:
If this is used in `anal.sh`, it might be useful in `eobs.sh` too?  @RussTreadon-NOAA thoughts?

Reply:
I don't know why all these settings are here for WCOSS2.  Most of these are not in my j-job drivers.    Are these settings from @GeorgeVandenberghe-NOAA  or @MichaelLueken-NOAA

Reply:
@RussTreadon-NOAA I took those settings from @MichaelLueken-NOAA's test scripts. I can adjust/remove these if they aren't needed...will need to retest analysis job without them to be sure.

Reply:
@KateFriedman-NOAA and @RussTreadon-NOAA, I made changes to my test script to assist the RTMA/URMA port.  You should use:

/lfs/h2/emc/da/noscrub/Michael.Lueken/GSI/scripts/rungsi1534L127_4denvar.sh_orig

or

export OMP_PLACES=cores
export OMP_NUM_THREADS=$threads
export OMP_STACKSIZE=1G
export FI_OFI_RXM_SAR_LIMIT=3145728

the rest aren't necessary.

Reply:
Thanks @MichaelLueken-NOAA, I have updated the anal block in WCOSS2.env based on your recommendations. Do any other DA jobs need those or similar settings for runtime? Thanks!

Reply:
@KateFriedman-NOAA, As far as I'm aware, those four environment variables should be all you need to run the GSI components.  The necessary environment variables for the EnKF are properly set for those components.

Reply:
Roger that, thanks @MichaelLueken-NOAA!